### PR TITLE
Add type annotations

### LINKED ISSUE #41: Missing Type Annotations
There are missing type annotations for some functions in this library.

The `typing` module does not exist on CircuitPython devices so the import needs to be wrapped in try/except to catch the error for missing import. There is an example of how that is done here:

```python
try:
    from typing import List, Tuple
except ImportError:
    pass
```

Once imported the typing annotations for the argument type(s), and return type(s) can be added to the function signature. Here is an example of a function that has had this done already:

```python
def wrap_text_to_pixels(
    string: str, max_width: int, font=None, indent0: str = "", indent1: str = ""
) -> List[str]:
```
If you are new to Git or Github we have a guide about contributing to our projects here: https://learn.adafruit.com/contribute-to-circuitpython-with-git-and-github

There is also a guide that covers our CI utilities and how to run them locally to ensure they will pass in Github Actions here: https://learn.adafruit.com/creating-and-sharing-a-circuitpython-library/check-your-code In particular the pages: `Sharing docs on ReadTheDocs` and `Check your code with pre-commit` contain the tools to install and commands to run locally to run the checks.

If you are attempting to resolve this issue and need help, you can post a comment on this issue and tag both @foamyguy and @kattni or reach out to us on Discord: https://adafru.it/discord in the `#circuitpython-dev` channel.

The following locations are reported by mypy to be missing type annotations:

- [ ] adafruit_motorkit.py:52
- [ ] adafruit_motorkit.py:70
- [ ] adafruit_motorkit.py:290

### PR DESCRIPTION
**Pull request by tekktrik**
Resolves #41

### CODE REVIEW

File: adafruit_motorkit.py [hunk 1]
```diff
@@ -50,8 +58,12 @@ class MotorKit:
     Alternately, if using with multiple I2C devices, you can specify the I2C bus."""
 
     def __init__(
-        self, address=0x60, i2c=None, steppers_microsteps=16, pwm_frequency=1600
-    ):
+        self,
+        address: int = 0x60,
+        i2c: Optional[I2C] = None,
+        steppers_microsteps: int = 16,
+        pwm_frequency: float = 1600,
```
Comment (FoamyGuy) [inline adafruit_motorkit.py:65]: Should the default value here be `1600.0`? Or should the type be int? It's listed as float as well inside of PCA9685 library so I'm guessing that is correct and that we'd want 1600.0 here to match.

File: adafruit_motorkit.py [hunk 2]
```diff
@@ -282,10 +296,10 @@ def stepper2(self):
...
 
     @property
-    def frequency(self):
+    def frequency(self) -> float:
         """The overall PCA9685 PWM frequency in Hertz."""
         return self._pca.frequency
 
     @frequency.setter
-    def frequency(self, pwm_frequency=1600):
+    def frequency(self, pwm_frequency: float = 1600) -> None:
```
Comment (FoamyGuy) [inline adafruit_motorkit.py:304]: same 1600.0 here I think.

**Review by FoamyGuy — CHANGES_REQUESTED** [review]
looks mostly good to me, but one question on `pwm_frequency` in the places where it's typed.

**Comment by tekktrik** [comment]
Yup, good point on the defaults being `int`, flipped 'em to `float`!

**Review by FoamyGuy — APPROVED** [review]
looks good to me. Thanks @tekktrik

**Pull request merged**